security/README.md

```diff
@@ -0,0 +1,19 @@
+
+
+# Network Security Policy
+
+Sample policy to allow two namesapces to talk; future version would be able to apply a wildcard like `devhub-*`. The operator will ensure the originating matches the source. The destination namespace will need to add a similar rule with the source and destination reversed to enable bi-directional communication.
+
+```yaml
+apiVersion: secops.pathfinder.gov.bc.ca/v1alpha1
```

Comment:
@stewartshea This line and the next one we'll need to get right before they policy goes live. I've assumed we'll just one one `SecOpsPolicy` CRD. We could use `NetSecOps` to be more clear that it is a networking policy and should be fed into Aporeto.

Reply:
I like secops as the overall main apiVersion here.